### ISSUE TITLE
Seek for the missing data file"genome_scan/chrEnds.txt"

### ISSUE THREAD

**Issue by ShirelyI** [speciationgenomics/data#2]
Hello Joana / Mark
I came across your excellent tutorial on analyzing speciationgenomics .
I am recently practising with a dataset you used in the course ,there seems to be some missing data files.
For example, I could not find "./genome_scan/chrEnds.txt" in the data section.
Could you please help with this file?
thanks
ShirelyI

**Comment by rturba** [comment]
Yeah, I would like to also see what that looks like. I just made assumptions of what it would be, but I'm getting some weird results, so it would be great to see if the issue is the file or the code. Thanks so much for this tutorial. It's been saving my life <3

**Comment by atengstedt** [comment]
Hi!
Stumbled upon this issue while I was searching for answers for another problem.
The file chrEnds.txt is just a simple file containing two tab delimited columns with a header, like below. 
The first column contains chromosome/scaffold names as in the data.
The second column is the length of each chromosome + however much space you want between each chromosome in the manhattan plot. In the below example I'm analyzing European eel. First chromosome is 88.06 Mb long, and I've added 5 Mb, to get a sufficient amount of white space before the next chromosome is plotted next to it.

chr  end
Chr_01  93060000
Chr_02  81640000
Chr_03  76680000
Chr_04  73140000
Chr_05  71100000
Chr_06  67030000
Chr_07  62210000
